mavros_msgs/GlobalPositionTarget goal_pos
---
float64 dist